Ruins/src/main/java/atomicstryker/ruins/common/FileHandler.java

```diff
@@ -176,18 +175,15 @@ RuinTemplate getTemplate(Random random, String biome)
     {
         try
         {
-            int rand = random.nextInt(vars.get(biome)[WEIGHT]);
-            int oldval = 0, increment = 0;
+            double rand = random.nextDouble()*vars.get(biome)[WEIGHT];

```

Comment:
formatting

Reply:
Not really an issue, formatting would leave it as "random.nextDouble() * vars.get(biome)[WEIGHT];"